.travis.yml

```diff
@@ -19,14 +19,32 @@ env:
         - COASTLINEDIR="$HOME/gmt-install-dir/coast"
         - PATH="$INSTALLDIR/bin:$PATH"
         - LD_LIBRARY_PATH="$INSTALLDIR/lib:$LD_LIBRARY_PATH"
+        - TRAVIS_EVENT_TYPE=cron
+        - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
+            export PYTHON=3.6
+            export CONDA_REQUIREMENTS="ci/requirements.txt"
```

Comment:
No need for a requirements file. You can set `CONDA_INSTALL_EXTRA="sphinx"` (see [here](https://github.com/fatiando/continuous-integration/blob/master/.travis.yml#L52)).